src/map/map.service.ts

```diff
@@ -0,0 +1,60 @@
+import { Injectable, Logger, HttpStatus } from '@nestjs/common';
+import { PrismaService } from '../prisma.service';
+import { ReadMapDto } from './dto/read-map.dto';
+import { HttpService } from '@nestjs/axios';
+import CustomException from 'src/exceptions/custom.exception';
+import { firstValueFrom } from 'rxjs';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class MapService {
+  constructor(
+    private prisma: PrismaService,
+    private readonly http: HttpService,
+    private readonly config: ConfigService,
+  ) {}
+
+  private readonly logger = new Logger(MapService.name);
+
+  async getMapById(
+    mapId: number,
+    lat: number,
+    lang: number,
+  ): Promise<ReadMapDto> {
+    try {
+      const map = await this.prisma.map.findUnique({
+        where: {
+          id: +mapId,
+        },
+      });
+
+      if (!map) {
+        throw new CustomException(HttpStatus.NOT_FOUND, 'Not Found - Map');
+      }
+
+      const defaultImage = this.config.get('DEFAULT_IMAGE');
+
+      const destinationLatitude = map.latitude;
+      const destinationLongitude = map.longitude;
```

Comment:
```suggestion
      const {
         latitude: destinationLatitude, 
         lognitude: destinationLongitude 
      } = map;
```
구조 분해 할당을 사용하는게 깔끔할 것 같습니다!
구조 분해시에도 이런식으로 이름 변경을 할 수 있습니다!

Reply:
오 좋습니다 반영할게요 !